tutorials/Python/12_ProteinAnalysis/12_ProteinAnalysis.ipynb

```diff
@@ -0,0 +1,468 @@
+    "# We make the ligand show up in a licorice representation\n",
+    "pdbview.add_representation('licorice', selection='MK1')\n",
+    "# We make the waters show up as red spheres\n",
+    "pdbview.add_representation('ball+stick', selection='water', color='red')\n",
+    "\n",
+    "# Finally we call the NGLView object to get an output\n",
+    "pdbview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Looking at temperature factors"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Up until now, we've done mostly things that could mostly be done by looking at the [PDB entry for 1HSG](https://www.rcsb.org/structure/1hsg). Let's apply these things to look at something that could be useful on a day to day basis.\n",
+    "\n",
+    "Here we will analyse the protein's  temperature factors (also known as bfactors) to know which parts of the protein are moving the most. If you want to know more about temperature factors, see [this useful guide by the PDB](https://pdb101.rcsb.org/learn/guide-to-understanding-pdb-data/dealing-with-coordinates)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Temperature factors are recorded in PDB files and are read by MDAnalysis when available. These can be found as an attribute of the `atoms` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Temperature factors of the protein residues\n",
+    "print(protein_residues.atoms.tempfactors)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Just printing the raw numbers isn't very informative.\n",
+    "What we can do here is plot the temperature factors of the alpha carbons in our protein.\n",
+    "\n",
+    "Do to this, let us first create a selection of the alpha carbon atoms (named \"CA\") for each chain:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Alpha carbons for chain A (also known as segid A)\n",
+    "chainA_alphaC = protein_residues.select_atoms('name CA and segid A')\n",
+    "# Alpha carbons for chain B (also known as segid B)\n",
+    "chainB_alphaC = protein_residues.select_atoms('name CA and segid B')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's use the plotting library matplotlib to create a plot of the alpha carbon temperature factors for each residue in each chain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We import pyplot from matplotlib\n",
+    "# Note the \"inline\" call is some jupyter magic to be able to show the plot\n",
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot\n",
```

Comment:
`from matplotlib import pyplot as plt` is very common. Should we use it here as well?

Reply:
Sure can do, it'll make them more ready for "python in the wild" 🙃